src/views/activityBar/changes/changeCommands.ts

```diff
@@ -12,3 +15,17 @@ export async function configureChangeLists(): Promise<void> {
 		`@ext:${EXTENSION_ID} changes`
 	);
 }
+
+export async function checkoutBranch({change}: {change: GerritChange}): Promise<void> {
```

Comment:
FYI the type of the argument that is passed to this is `ChangeTreeView`. You don't really need to update it but might be handy to know what data you have access to.